types/dns-packet/index.d.ts

```diff
@@ -0,0 +1,173 @@
+// Type definitions for dns-packet 5.2
+// Project: https://github.com/mafintosh/dns-packet
+// Definitions by: John Hurliman <https://github.com/jhurliman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
```

Comment:
I believe we only accept declarations to be allowed under the MIT license - the same as the rest of the repo. Maybe @RyanCavanaugh can clarify.

Reply:
Fixed, thanks for the catch